notebooks/263-stable-diffusion-torchdynamo-backend/263-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q "torch==2.0.1"
</pre><p>does torch 2.1 not supported?</p><p>we prefer do not use strict dependencies if there is no reason for that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>